### DISCUSSION TITLE
Where are the config files on windows?

### DISCUSSION THREAD

**Discussion by Aurelynn** [legendary-gl/legendary#286] [Q&A] [answered]
Spent forever looking for them in all 3 appdata locations and registry and can't find any config files. Legendary keeps trying to login using wrong credentials.

**Comment by flavioislima** [comment]
In your Home folder, in a folder called `.config/legendary.`

**Reply by Aurelynn** [reply]
Thank you, I would never have looked in the root of user folder.

**Comment by derrod** [comment, marked as answer]
It's in `%USERPROFILE%\.config\legendary`. You can also use `legendary status` to see the current config directory.

**Reply by Aurelynn** [reply]
Thank you, I would never have looked here.

**Comment by vdias** [comment]
Can the .config folder be moved to another folder?
